include/psa/crypto.h

```diff
@@ -2527,9 +2527,11 @@ psa_status_t psa_aead_set_lengths(psa_aead_operation_t *operation,
  *
  * Additional data is authenticated, but not encrypted.
  *
- * You may call this function multiple times to pass successive fragments
- * of the additional data. You may not call this function after passing
- * data to encrypt or decrypt with psa_aead_update().
+ * Unless you are using a GCM algorithm you may call this function multiple
```

Comment:
This limitation only applies to the software implementation, not to drivers. Please retain the original wording and make a separate `\note` for the temporary limitation of the built-in implementation.

Another temporary limitation of the built-in implementation is that CCM is not yet supported.